### DISCUSSION TITLE
About gRPC, RabbitMQ, Kafka

### DISCUSSION THREAD

**Discussion by rhadamez** [adonisjs/core#1679] [Ideas]
Hey!

Have you think about integrate these tecs to Adonis by default? I think it would be a great differencial.

Cheers!

**Comment by DraftProducts** [comment]
I'm using RabbitMQ as Provider in Adonis V5, it's working well, you should check it !
I'm not sure it's a nice iead to integrete it by default, thoose tecs are realy specifics for big projects so the majority of developers will never use it and it will only make Adonis heavier while it's easy to use Providers: 
https://preview.adonisjs.com/guides/directory-structure#the-providers-directory

I haven't seen any documention about it, but it's typescript :)
